test/interpreter_test.src

```diff
@@ -214,6 +216,8 @@ function test_while() {
     return 1;
 }
 function b9main() {
```

Comment:
This is not the right place to add the calls to test_mul/test_div. You would add it to the list in b9test.cpp